### PR TITLE
Add a crawler for Grimaldi's Pizzeria

### LINKED ISSUE #3519: Grimaldis Pizzeria
https://www.grimaldispizzeria.com/locations/

### PR DESCRIPTION
**Pull request by coffeesippa**
Added a crawler for Grimaldi's Pizzeria.

Fixes #3519

### CODE REVIEW

**Review by iandees — COMMENTED** [review]
Please use the `black` linter to format your code.

Also, were you unable to find lat/lon values on the page?

File: locations/spiders/grimaldis_pizzeria.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    def parse(self, response):
+        locations = response.css('div.location_block')
+
+        for loc in locations:
+            address = self._format_address(loc.css('.store_address::text').getall())
+
+            properties = {
+                "ref": loc.css('h3.loc_title::text').get(),
+                "name": loc.css('.loc_title::text').get(),
+                "addr_full": address.street_address,
```
Comment (Cj-Malone) [inline locations/spiders/grimaldis_pizzeria.py:23]: This should be "street_address", not "addr_full"

Comment (coffeesippa) [reply locations/spiders/grimaldis_pizzeria.py:34]: I get a Key error when I try to use `street_address` for the key value. 
`KeyError: 'GeojsonPointItem does not support field: street_address'`

Looks like the allowed keys for **GeojsonPointItem** are:
*`addr_full`
* `street` (I assume this must be used in combination with `housenumber`)

**Comment by coffeesippa** [comment]
Thanks @iandees. I was able to find the lat/lon values buried elsewhere on the page. Putting the changes up for review now.

File: locations/spiders/grimaldis_pizzeria.py (hunk 1 above)
Comment (iandees) [reply locations/spiders/grimaldis_pizzeria.py:34]: Make sure your fork is up to date. We added the street_address field a week ago, and it looks like your fork is missing that change.

Comment (coffeesippa) [reply locations/spiders/grimaldis_pizzeria.py]: Yup, that was the issue. Fixed! Though it looks like updating the fork brought in a lot of extra change into the MR. Let me know if you'd like me to try to fix that somehow.

**Comment by iandees** [comment]
Thanks for doing that! I rebased your changes onto the `master` branch so that it only contained your changes and pushed that over here https://github.com/alltheplaces/alltheplaces/pull/3610.

**Pull request closed without merging**